roles/confluent.kafka/templates/server.properties.j2

```diff
@@ -1,7 +1,7 @@
 # Maintained by Ansible
-listeners=PLAINTEXT://:{{broker.config.port}}
+listeners=PLAINTEXT://:{{kafka_port}}
```

Comment:
Recommend leaving in 'broker' as kafka is not a component but the platform name.